en/Getting started/Sync your notes across devices.md

```diff
@@ -0,0 +1,111 @@
+
+- Dropbox
+- Google Drive
+- OneDrive
+- Syncthing
+
+### iCloud Drive
+
+Obsidian can use iCloud Drive as a local file system.
+
+#### Create a new vault in iCloud Drive
+
+To create a new vault in iCloud Drive on your iPhone or iPad:
+
+1. Tap **Create new vault**.
+2. In **Vault name**, enter the name of your vault.
+3. Enable **Store in iCloud**.
+4. Tap **Create**.
+
+Obsidian has created a new folder inside your iCloud Drive. To open the iCloud Drive folder as a vault on your computer:
+
+1. Open **Obsidian** on your computer.
+2. In the the vault switcher, to the right of **Open folder as vault**, click **Open**.
+3. Navigate to **iCloud Drive** -> **Obsidian**.
+4. Select the folder with the name of the vault you want to sync with.
+
+#### Sync an existing vault with iCloud Drive
+
+To sync an existing vault using iCloud, you need to create an empty vault in iCloud Drive and then move your notes on your other device to the empty vault.
+
+To create a new empty vault in iCloud Drive:
+
+1. Tap **Create new vault**.
+2. In **Vault name**, enter the same name as the vault you want to sync with.
+3. Enable **Store in iCloud**.
+4. Tap **Create**.
+
+To move your notes to the new vault in your iCloud Drive:
+
+1. On your computer, open the **iCloud Drive** folder.
+2. Open the **Obsidian** folder. It might take a few minutes to appear.
+3. Move the files in your existing vault to the folder with the name of your vault.
+
+iCloud syncs the files to your mobile device. Depending on the size of your vault, this might take a few minutes.
+
+### Working Copy
+
+If you store your notes in a [Git](https://git-scm.com/) repository, you may want to look at [Working Copy](https://apps.apple.com/us/app/working-copy-git-client/id896694807)—a Git client for your iOS.
+
+To sync using Working Copy:
+
+1. Create an empty vault on your iPhone or iPad.
+2. Use the **Setup Folder Sync** action and select your empty vault.
+3. Commit and push any changes to your vault using the Working Copy app.
+
+**Note:** While we don't officially support this method, several of our users have reported that they've successfully used Working Copy to sync their notes.
+
+### Why can't I sync using X?
+
+We understand that many of you use other services for syncing files and you'd prefer to use it for syncing your notes as well.
+
+Obsidian works differently from other Markdown editors on iOS. Editors such as 1Writer and iA Writer access one note at a time, which lets them use built-in support for documents.
+
+In contrast, many features in Obsidian need access to your entire vault. For example if you rename a file, then Obsidian needs to update all files in the vault that links to that file.
+
+Since there's no support in iOS for accessing many files at once, rather than one at a time, supporting other sync services becomes a major challenge. Although we'll continue to evaluate available options, they're limited at the moment.
```

Comment:
Hmm I think this isn't technically accurate. I believe this is possible, it's just that we'll need to make a pretty major change to the way we're using the file system API on iOS right now, which is something we haven't yet been able to spend a lot of time with.

Last I've experimented with it, the way to request permission externally was fairly tedious, especially across the native bridge, which is why we haven't really implemented this.

I believe it might be possible to have this implemented in the future, unless we run into technical challenges when attempting to use the proper API on iOS (which historically has been a common occurrence...).

Reply:
I would consider rephrasing this part to include a bit less details, since the situation here isn't as clear-cut as it is, and providing this kind of explanation doesn't hold to scrutiny by users who are familiar with Apple's ecosystem.

I'd propose something in the words of "Implementing a system to read, modify, and monitor an entire folder structure comprising of possibly thousand of notes outside of the currently supported locations is challenging. We will attempt to overcome this limitation in the future."